content/css/concepts/display/display.md

```diff
@@ -214,3 +214,5 @@ Setting a `div` element to behave like a flex box.
   display: flex;
 }
 ```
+
```

Comment:
Add in a sentence that leads the above code block to this output. Something like this:
"This does [XYZ] and results in:"